mdaviz/mda_folder_table_view_pager.py

```diff
@@ -0,0 +1,102 @@
+from functools import partial
+
+from PyQt5 import QtCore, QtWidgets
+
+class _AlignCenterDelegate(QtWidgets.QStyledItemDelegate):
+    """https://stackoverflow.com/a/61722299"""
+
+    def initStyleOption(self, option, index):
+        super().initStyleOption(option, index)
+        option.displayAlignment = QtCore.Qt.AlignCenter
+        
+class mdaFolderTableView(QtWidgets.QWidget):
+    # ui_file = utils.getUiFileName(__file__)  # WARNING no ui file, just a tab in mda_folder_search.ui
```

Comment:
I'm confused about this.  This is a widget, not just a pager, right?

Reply:
same comment as above.